### ISSUE TITLE
first US

### ISSUE THREAD

**Issue by JonasDaJones** [JonasDaJones/JonasDaJones#2]

**Comment by julialoeschel** [comment]
Hey Jonas. 
Super story, gut überlegt!
mir fehlt noch die Datenerstellung bei den Tasks. 

Und du solltest dir evt jetzt schon Gedanken machen,  wie die horizontale Scroll funktion aussehen soll. Da gibt es viele Varianten, es lohnt sich das Internet jetzt schonmal zu befragen, was sich da am besten eignet :) 
(oder du nimmst bewusst die simpelste, und erstellst in einer späteren US dann eine fancy slider funktion ;) )

**Comment by julialoeschel** [comment]
🎉

**Comment by JonasDaJones** [comment]
Ja, ich hatte mir gedacht, ersteinmal eine simple flexbox mit x overflow zu gestalten und vielleicht später einen snap effekt hizuzufügen.
